code/datums/mutations/speech.dm

```diff
@@ -102,10 +100,7 @@
 		message = replacetext(message," fat "," thin ")
 		message = replacetext(message," beer "," water with ice ")
 		message = replacetext(message," drink "," water ")
-		message = replacetext(message," feminist "," empowered woman ")
 		message = replacetext(message," i hate you "," you're mean ")
-		message = replacetext(message," nigger "," african american ")
-		message = replacetext(message," jew "," jewish ")
```

Comment:
Keep this too

Reply:
Your sense of humor never ceases to amuse me.

Reply:
Replacing jew with jewish doesn't seem particularly relevant- I don't really see an issue with it?